packages/rich-text/README.md

```diff
@@ -356,6 +358,10 @@ _Returns_
 
 -   `RichTextValue`: A new value with replacements applied.
 
+### RichTextValue
+
+Undocumented declaration.
```

Comment:
Unsure what to do about this. `RichTextValue` is an exported type.

Reply:
I think we can just add some inline docs where declaring `RichTextValue` and the docs script should be able to pick them up.